changelog.d/10077.feature

```diff
@@ -0,0 +1 @@
+Make reason and score parameters optional for reporting content. Implements MSC2414. Contributed by Callum Brown.
```

Comment:
Can you link to the MSC in there? Changelog files will be compiled into a markdown document on release so feel free to make it a markdown link.

Reply:
Sure, didn't realise it was markdown.

Reply:
To be fair we should probably spell it out better.